src/common/data/stores/app/index.ts

```
@@ -51,6 +52,18 @@ const makeStoreFunc: MatativeConfig<AppStore> = (set, get, state) => ({
     get().space.clear();
     localStorage.removeItem(LOCAL_STORAGE_LOCATION);
   },
+  clearLocalSpaces: () => {
+    // remove only the space slice while keeping the rest of the persisted data
+    get().space.clear();
+    try {
+      const raw = localStorage.getItem(LOCAL_STORAGE_LOCATION);
+      if (raw) {
+        const persisted = JSON.parse(raw);
+        delete persisted.space;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)